stdlib/public/core/CTypes.swift

```diff
@@ -227,6 +227,28 @@ extension UInt {
   }
 }
 
+#if arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS) || os(Windows))
+@_fixed_layout
+public struct CVaListPointer {
+  @_versioned // FIXME(sil-serialize-all)
```

Comment:
Nitpick: we now have non-underscored forms of `_versioned` and `_inlineable` (`usableFromInline` and 'inlinable`, respectively). I know this is an artifact of how long this patch has sat here, though!